cmake/BuildLibXXhash.cmake

```diff
@@ -0,0 +1,21 @@
+macro(libxxhash_build)
+    set(xxhash_src third_party/xxHash/xxhash.c)
+
+    if (CC_HAS_WNO_IMPLICIT_FALLTHROUGH)
+        set_source_files_properties(${xxhash_src}
+                PROPERTIES COMPILE_FLAGS
+                -Wno-implicit-fallthrough)
+    endif()
+
+    # Specify "XXH_NAMESPACE" to prevent symbol clash with zstd
+    # that also uses xxhash.
+    set_source_files_properties(${xxhash_src}
+            PROPERTIES COMPILE_FLAGS "-Ofast -DXXH_NAMESPACE=tnt_")
```

Comment:
Why did you chose `-Ofast`?

Reply:
Because zstd is built with such flag https://github.com/tarantool/tarantool/blob/02a7f9af2c83a62e871d7a8e424ed05f8b453ae0/cmake/BuildZSTD.cmake#L34

Reply:
It seems, everything was resolved.